1-js/02-first-steps/09-alert-prompt-confirm/article.md

```diff
@@ -1,109 +1,109 @@
-# Interaction: alert, prompt, confirm
+# Interakcje: alert, prompt, confirm
 
-In this part of the tutorial we cover JavaScript language "as is", without environment-specific tweaks.
+W tej części poradnika przedstawimy język JavaScript w podstawowej formie, bez żadnych ulepszeń dla danego środowiska.
 
-But we'll still be using the browser as our demo environment, so we should know at least a few of its user-interface functions. In this chapter, we'll get familiar with the browser functions `alert`, `prompt` and `confirm`.
+Nadal będziemy używać przeglądarki jako domyślnego środowiska, więc powinniśmy znać przynajmniej kilka funkcji interfejsu przeglądarki. W tym rozdziale, zaznajomimy się z funkcjami dostępnymi w przeglądarce: `alert`, `prompt` and `confirm`.
```

Comment:
Co sądzisz o:
`Nadal będziemy używać przeglądarki jako domyślnego środowiska, więc powinniśmy znać przynajmniej kilka funkcji jej interfejsu`
Słowo `przeglądarki` pojawia się zbyt często.

Reply:
Zdecydowanie lepiej to brzmi, dzięki!